Directory.Build.props

```diff
@@ -8,6 +8,7 @@
     <LangVersion>latest</LangVersion>
     <Nullable>enable</Nullable>
     <AnalysisMode>AllEnabledByDefault</AnalysisMode>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>

```

Comment:
Had to ![image](https://user-images.githubusercontent.com/3275797/141462478-63f03216-2cb3-41b6-b271-eb52e3264104.png) to learn this wasn't all about "end-of-line".

Reply:
end of line? I thought it was end of life